lib/click-handler.js

```diff
@@ -50,47 +49,45 @@ function tryLoad(urls, cb) {
 }
 
 function getResolverUrls(dataAttr) {
-  const resolverStrings = dataAttr.resolver.replace(/\s/g, '').split('|');
-
-  const urls = resolverStrings.map((resolverName) => {
-    const resolver = resolverHandlers.get(resolverName);
-    if (resolver) {
-      chrome.runtime.sendMessage({
-        type: 'track',
-        payload: {
-          category: 'resolver',
-          action: 'invoke',
-          label: resolverName,
-        },
-      });
-
-      return [].concat(resolver(dataAttr)).map((url) => {
-        if (!url) {
-          return null;
-        }
-
-        if (typeof url === 'object') {
-          return {
-            url: url.url.replace('{BASE_URL}', 'https://github.com'),
-            method: url.method,
-          };
-        }
-
-        if (url.startsWith('{BASE_URL}')) {
-          return {
-            url: url.replace('{BASE_URL}', 'https://github.com'),
-          };
-        }
-
-        return {
-          url: `https://githublinker.herokuapp.com/ping?url=${url}`,
-          method: 'GET',
-        };
-      });
-    }
+  const pluginName = dataAttr._pluginName;
+  const resolver = pluginManager.getClickHandler(pluginName);
```

Comment:
I think we should rename `getClickHandler` to `getResolver`, since that more accurately describes what the return value does (and matches the variable name used here).